prime-angular-frontend/src/app/modules/paper-enrolment/pages/obo-sites-page/obo-sites-overview.component.ts

```diff
@@ -0,0 +1,111 @@
+
+      <button *ngIf="true"
+              mat-icon-button
+              matTooltip="Edit Job Site Information"
+              (click)="onRoute(PaperEnrolmentRoutes.OBO_SITES)">
+        <mat-icon>edit</mat-icon>
+      </button>
+    </app-page-subheader>
+
+    <ng-container *ngFor="let careSetting of careSettings">
+
+      <ng-container *ngFor="let oboSite of oboSites">
+        <ng-container *ngIf="oboSite.careSettingCode === careSetting.careSettingCode">
+          <app-enrollee-property title="Care Setting"
+                                 [makeBold]="true">
+            <div class="mb-3">{{ careSetting.careSettingCode | configCode: 'careSettings' }}
+              <span *ngIf="oboSite.careSettingCode === healthAuthority">
+                  ({{ oboSite.healthAuthorityCode | configCode: 'healthAuthorities' | capitalize: true }})
+                </span>
+            </div>
+
+            <app-enrollee-property *ngIf="oboSite.careSettingCode !== healthAuthority"
+                                   title="Site Name"
+                                   [makeBold]="true">
+              {{ oboSite.siteName | default }}
+            </app-enrollee-property>
+
+            <app-enrollee-property *ngIf="oboSite.careSettingCode === healthAuthority"
+                                   title="Facility Name"
+                                   [makeBold]="true">
+              {{ oboSite.facilityName | default }}
+            </app-enrollee-property>
+
+            <app-enrollee-property *ngIf="oboSite.careSettingCode !== healthAuthority"
+                                   title="Site ID"
+                                   [makeBold]="true">
+              {{ oboSite.pec | default }}
+            </app-enrollee-property>
+
+            <app-enrollee-property title="Job Title"
+                                   [makeBold]="true">
+              {{ oboSite.jobTitle | default }}
+            </app-enrollee-property>
+
+            <app-enrollee-property title="Site Address"
+                                   [makeBold]="true">
+              <app-enrollee-property title="Street">
+                {{ oboSite.physicalAddress?.street | default }}
+              </app-enrollee-property>
+
+              <app-enrollee-property title="City">
+                {{ oboSite.physicalAddress?.city | default }}
+              </app-enrollee-property>
+
+              <app-enrollee-property title="Province">
+                {{ oboSite.physicalAddress?.provinceCode | configCode: 'provinces' | default }}
+              </app-enrollee-property>
+
+              <app-enrollee-property title="Postal Code">
+                {{ oboSite.physicalAddress?.postal | postal | default }}
+              </app-enrollee-property>
+
+            </app-enrollee-property>
+          </app-enrollee-property>
+        </ng-container>
+      </ng-container>
+    </ng-container>
+
+    </app-page-section>
+  `,
+  styles: [
```

Comment:
If you look at the examples in HealthAuthority you can see I've added ViewEncapsulation of None to the overview components